From: gregkh@suse.de
Subject: [PATCH] USB: fix up compiler warnings when CONFIG_USB_DEBUG is not set.

[PATCH] USB: fix up compiler warnings when CONFIG_USB_DEBUG is not set.

As pointed out by Kay Sievers <kay.sievers@vrfy.org>

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/core/hub.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/usb/core/hub.c b/drivers/usb/core/hub.c
--- a/drivers/usb/core/hub.c	2005-03-08 16:40:27 -08:00
+++ b/drivers/usb/core/hub.c	2005-03-08 16:40:27 -08:00
@@ -1110,7 +1110,7 @@
 }
 
 #else
-static inline void show_string(struct usb_device *udev, char *id, int index)
+static inline void show_string(struct usb_device *udev, char *id, char *string)
 {}
 #endif
 
